### PR TITLE
build fixes when using install-glib and on openbsd

### PR DESCRIPTION
**Pull request by ailin-nemui**

### CODE REVIEW

**Comment by ailin-nemui** [comment]
these are fixes in an order -only dependency that was missed (had to be manually added now) due to fix #1166 and automatic configuration of some options for building glib on openbsd

**Comment by ailin-nemui** [comment]
@irssi/developers

File: meson.build [hunk 1]
```diff
@@ -162,15 +163,70 @@ if not glib_dep.found()
...
+
+  glib_internal_dependencies = [
+    dependency('threads'),
+  ]
+  glib_internal_configure_args = []
+
+  glib_internal_local_lib = false
+  if cc.has_function('iconv_open')
+    # pass
+  else
```
Comment (dequis) [inline meson.build:181]: Wait does this language not support something like `if not expr`?

Comment (ailin-nemui) [reply meson.build]: actually.. it does. fixed

**Comment by dequis** [comment]
I guess it's fine!

Why do we even have an install-glib, anyway?

**Comment by ailin-nemui** [comment]
good question, maybe to work around #1180 with -Dinstall-glib=force

**Comment by dequis** [comment]
IIIiii really don't think we should be in the business of fixing issues with our dependencies that distros don't.

**Comment by dequis** [comment]
To answer my own question: the oss-fuzz build script downloads and statically links glib https://github.com/google/oss-fuzz/blob/master/projects/irssi/build.sh#L37

**Comment by ailin-nemui** [comment]
oh, good to know!

**Pull request merged**